src/main/java/com/auth0/client/mgmt/UsersEntity.java

```diff
@@ -337,6 +337,35 @@ public Request deleteMultifactorProvider(String userId, String provider) {
         return request;
     }
 
+    /**
+     * Links two User's Identities via token using the link_with parameter.
+     * A token with scope update:users is needed or update:current_user_identities if access token is for same user.
```

Comment:
According to the API docs, the `link_with` approach will only work for `update:current_user_identities` scoped tokens. That also means that the API client would need to be set up with that token. I think the use case of these scopes and particularly this endpoint is more related to native clients. But I don't have any issues with adding it. 

Let's fix that doc

```suggestion
     * A token with scope update:current_user_identities is needed. It only works for the user the access token represents.
```

Reply:
Added. No problem!